srml/authority-discovery/src/lib.rs

```diff
@@ -166,6 +166,10 @@ mod tests {
 		}
 	}
 
+	parameter_types! {
+		pub const DisabledValidatorsThreshold: Perbill = Perbill::from_percent(33);
```

Comment:
why 33 here?

Reply:
I'ts only a test, I don't think it matters at all.